server/mainPr/src/main/java/com/team23/mainPr/Profile/Service/ProfileService.java

```diff
@@ -0,0 +1,69 @@
+package com.team23.mainPr.Profile.Service;
+
+import com.team23.mainPr.Profile.Dto.ProfileUpdateDto;
+import com.team23.mainPr.Profile.Entity.Profile;
+import com.team23.mainPr.Profile.Repository.ProfileRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+/*
+* refactor : 커스텀 예외 클래스 만들기
+*
+* */
+public class ProfileService {
+
+    /*
+    * description : 프로필 내용 조회
+    * refactor : 커스텀 예외 클래스 생성으로 오류 상황에 대해서 친절한 응답 만들기
+    * */
+    @Autowired
+    ProfileRepository profileRepository;
+    public Profile getProfile(Integer postId) {
```

Comment:
```suggestion
    @Autowired
    ProfileRepository profileRepository;
    
    public Profile getProfile(Integer postId) {
```

기본적인 컨벤션을 준수해주시기 바랍니다.